scripts/getModelFromSource.js

```diff
@@ -100,14 +71,14 @@ function resolveImports(rules, opts) {
 						`La règle '${ruleName}' n'existe pas dans ${value[fromKeyword]}`
 					)
 				}
-				const traversedRules = getDependencies(engine, rule)
 				const updatedRawNode = { ...rule.rawNode, ...attrs }
-				traversedRules.push([ruleName, updatedRawNode])
-				checkMemoToIgnoreTraversedRules(traversedRules, modifiedRulesMemo)
-				acc.push(...traversedRules)
-				if (updatedRawNode !== rule.rawNode) {
-					modifiedRulesMemo.push(ruleName)
-				}
+				acc.push([ruleName, updatedRawNode])
```

Comment:
Ce push est pas en trop ? Je suis pas sur de bien comprendre la simplification, on push 2 fois dans acc

Reply:
> Ce push est pas en trop @EmileRolley ? Je suis pas sur de bien comprendre la simplification, on push 2 fois dans acc

Hmm, à première vue, je ne vois pas le push doublon.. La simplification était dans le fait de ne pas avoir à stocker dans une liste parallèle les règles déjà parcourue est éviter de répéter la récupération des dépendances.

Reply:
> D'ailleurs, j'ai remarqué qu'il y a des doublons de règles dans ruleDeps, je sais pas si c'est logique

Il ne devrait pas avoir de doublons normalement, je regarde ça demain matin :+1:

Reply:
Fix in https://github.com/datagir/nosgestesclimat/pull/1815/commits/c0980130185cb9ffc185c0742a0960507dd037e7